src/pudl/helpers.py

```diff
@@ -2062,12 +2068,18 @@ def get_dagster_execution_config(
                 },
             },
         }
+
+    start_method_config = {}
+    if "forkserver" in multiprocessing.get_all_start_methods():
+        start_method_config = {"forkserver": {"preload_modules": ["pudl"]}}
```

Comment:
This saves like 9s per process which matters when we have hundreds of checks that only take a couple seconds to actually run.